contracts/src/CeramicAnchorServiceV2.sol

```diff
@@ -17,15 +17,24 @@ contract CeramicAnchorServiceV2 is Ownable {
 
     //upon successful anchor
     event DidAnchor(address indexed _service, bytes _root);
+    // event DidAnchor(address indexed _service, bytes32 _root);
```

Comment:
dev comments.

In the future, please, make sure to not leave dev artefacts like this in the PRs. It is quite annoying.